### PR TITLE
EUI-8718 Register other org - PBA number

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8718

### Change description ###
Register other org - (Display) Does your organisation have a PBA number?

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/register-org/components/document-exchange-reference-details/document-exchange-reference-details.component.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export class DocumentExchangeReferenceDetailsComponent extends RegisterComponent
   public dxFormGroup: FormGroup;
 
   constructor(public readonly router: Router,
-    public readonly registerOrgService: RegisterOrgService,
+    public readonly registerOrgService: RegisterOrgService
```
Comment (Daniel-Lam) [inline src/register-org/components/document-exchange-reference-details/document-exchange-reference-details.component.ts:15]: @johnbenjamin-hmcts - does this service need to be `public`? Same for the router injection above

File: src/register-org/components/document-exchange-reference/document-exchange-reference.component.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ export class DocumentExchangeReferenceComponent extends RegisterComponent implem
   public dxError: ErrorMessage;
 
   constructor(public readonly router: Router,
-    public readonly registerOrgService: RegisterOrgService,
+    public readonly registerOrgService: RegisterOrgService
```
Comment (Daniel-Lam) [inline src/register-org/components/document-exchange-reference/document-exchange-reference.component.ts:19]: @johnbenjamin-hmcts - as above on whether this should be `public`

File: src/register-org/components/payment-by-account/payment-by-account.component.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  templateUrl: './payment-by-account.component.html'
+})
+export class PaymentByAccountComponent extends RegisterComponent implements OnInit, OnDestroy {
+  @ViewChild('errorSummaryTitleElement') public errorSummaryTitleElement: ElementRef;
+
+  public pbaFormGroup: FormGroup;
+  public pbaError: ErrorMessage;
+
+  constructor(public readonly router: Router,
+    public readonly registerOrgService: RegisterOrgService
```
Comment (Daniel-Lam) [inline src/register-org/components/payment-by-account/payment-by-account.component.ts:19]: @johnbenjamin-hmcts - same thing as above, on use of `public`

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - just a few minor observations

File: src/register-org/components/document-exchange-reference-details/document-exchange-reference-details.component.ts (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/register-org/components/document-exchange-reference-details/document-exchange-reference-details.component.ts:15]: @Daniel-Lam - All the components extend from the master page component called `RegisterComponent` that takes in Router and RegisterOrgService as the constructor properties.

Having them as private on both the master page component and child components did not work.

File: src/register-org/components/payment-by-account/payment-by-account.component.ts (hunk 3 above)
Comment (johnbenjamin-hmcts) [reply src/register-org/components/payment-by-account/payment-by-account.component.ts:19]: @Daniel-Lam - All the components extend from the master page component called `RegisterComponent` that takes in Router and RegisterOrgService as the constructor properties.

Having them as private on both the master page component and child components did not work.

**Pull request merged**